lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc

```diff
@@ -134,16 +143,22 @@ function dosomething_campaign_preprocess_common_vars(&$vars, &$wrapper) {
         ),
       ),
       'twitter' => array(
```

Comment:
@angaither sounds good, I'll update and remove those changes in `lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc`. Thanks for the github help to do that :) 

Yes - I was trying to add changes to the content [here](https://github.com/DoSomething/phoenix/blob/72e94a3639cff8c75cd167118975a59b591904b8/lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc#L210) as well but that is actually a string and not an array so would be difficult to inject to the end without a set pattern (each campaign will have a different amount of bullet points, the content here might not be bullet points etc.). 

Unless, we want this to only be specific for the organ donation campaign, I can do a check to see if the organ donation module is enabled, add a bullet point at the end of this string?